index.html

```diff
@@ -3572,7 +3572,7 @@ <h2>Definition of Roles</h2>
 			<div class="role-description">
 				<p>A set of user interface <a>objects</a> that is not intended to be included in a page summary or table of contents by <a>assistive technologies</a>.</p>
 				<p>Contrast with <rref>region</rref>, which is a grouping of user interface objects that will be included in a page summary or table of contents.</p>
-				<p>Authors SHOULD use a <code>group</code> to form a logical collection of items in a <a>widget</a>, such as children in a tree widget forming a collection of siblings in a hierarchy, or a collection of items having the same container in a directory. However, when a <code>group</code> is used in the context of a <rref>list</rref>, authors MUST limit its children to <rref>listitem</rref> elements. Therefore, proper handling of <code>group</code> by authors and assistive technologies is determined by the context in which it is provided.</p>
+				<p>Authors SHOULD use a <code>group</code> to form a logical collection of items in a <a>widget</a>, such as children in a tree widget forming a collection of siblings in a hierarchy, or a collection of items having the same container in a directory. However, when a <code>group</code> is used in the context of a <rref>listbox</rref>, authors MUST limit its children to <rref>option</rref> elements. Therefore, proper handling of <code>group</code> by authors and assistive technologies is determined by the context in which it is provided.</p>
```

Comment:
Remove:

>, or a collection of items having the same container in a directory

A directory is a list so we can't have groups inside directories either. Note that the directory definition confusingly uses the word group where it should not. But, that should be addressed separately.

Reply:
I've removed the noted text.  Thank you @mcking65